matflow_defdap/main.py

```diff
@@ -87,7 +87,6 @@ def load_microstructure_EBSD():
 def write_param_file_2(path, EBSD, root_path, scaling_factor):
     obj = {
         'ebsd_filename': EBSD['filename'],
-        'ebsd_flip_vert': EBSD.get('flip_vert', False),
```

Comment:
I'm guessing there is a reason `ebsd_flip_vert` has been removed?

Reply:
I double-checked that this difference is between the `master` and the `dev` branch, and I cloned the whole package from the `master`. Shall I redo this based on the `dev` branch?